### PR TITLE
update contribution guide

### PR DESCRIPTION
**Pull request by ncbaratta**

### CODE REVIEW

File: topics/contributing_to_the_framework.adoc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [[ccg-contributing-to-the-framework]]
-== Contributing to the framework
+== Contributing to the guide
```
Comment (pmkovar) [inline topics/contributing_to_the_framework.adoc]: +1!

File: topics/contributing_to_the_framework.adoc [hunk 2]
```diff
@@ -51,11 +51,24 @@ We encourage contributions to the _Community Collaboration Guide_. In particular
 
 * You must have downloaded, installed, and configured Git on your desktop.
 
-* Ensure that your configuration gives you the ability to manage files locally by using the command line.
+* Ensure that you have a xref:ccg-fork-the-upstream-repository[copy of the repository on your local system].
 
-.Related information
+* You need the appropriate permissions to create pull requests. Request permission by emailing your GitHub username to ccs-upstream@redhat.com.
```
Comment (pmkovar) [inline topics/contributing_to_the_framework.adoc]: Is this true, though? I think anybody with a fork can create a pull request on GitHub.

File: topics/contributing_to_the_framework.adoc [hunk 3]
```diff
@@ -51,11 +51,24 @@ We encourage contributions to the _Community Collaboration Guide_. In particular
...
+
+.Procedure
+
+. Navigate to the https://github.com/redhat-documentation/community-collaboration-guide/issues[*Issues*] tab and click on the `Title` of the issue you'd like to address.
+
+. Assign the issue to yourself so that others know you are working on it.
+
+. Keep the issue up to date with your progress
+
+. Submit a PR with your edits
```
Comment (pmkovar) [inline topics/contributing_to_the_framework.adoc]: Might be better to expand PR here.

File: topics/contributing_to_the_framework.adoc (hunk 2 above)
Comment (ncbaratta) [reply topics/contributing_to_the_framework.adoc]: You know, I don't know.  I thought folks needed permission ... let me see if I can get a new writer to give it a try without permission.

File: topics/contributing_to_the_framework.adoc (hunk 3 above)
Comment (ncbaratta) [inline topics/contributing_to_the_framework.adoc]:
```suggestion
. Submit a Pull Request (PR) with your edits
```

File: topics/contributing_to_the_framework.adoc (hunk 2 above)
Comment (ncbaratta) [reply topics/contributing_to_the_framework.adoc]: I just checked and the repository is set so anyone can 'Read' - so they can create forks, but not push (create pull requests).

Comment (pmkovar) [reply topics/contributing_to_the_framework.adoc]: Sure, they can't push PRs or commits to the base repo, but they can create a PR that others with permissions can push, I believe.

I remember this discussion in the GNOME project about mirroring repos on GitHub that there is no way to disable forking and submitting PRs, which makes the mirroring a bit confusing.  

Seems like this is a feature GitHub won't change, see https://github.com/dear-github/dear-github/issues/84.

Comment (ncbaratta) [reply topics/contributing_to_the_framework.adoc]: Thanks for clarifying!

File: topics/contributing_to_the_framework.adoc [hunk 4]
```diff
@@ -16,7 +16,7 @@ Currently, the best method for proposing a change to the _Community Collaboratio
 
 .Prerequisites
 
-You must be a member of the GitHub `community-collaboration-guide` repository.
+You must be a member of the GitHub `community-collaboration-guide` repository. Request to join by emailing your GitHub username to ccs-upstream@redhat.com.
```
Comment (ncbaratta) [inline topics/contributing_to_the_framework.adoc]: @pmkovar is this right?

Comment (pmkovar) [reply topics/contributing_to_the_framework.adoc]: It is technically correct, yes, even though once we clarify the license in #119 we could start accepting drive-by contributions (think: typo fixes) without the need to become a member first.

Comment (ncbaratta) [reply topics/contributing_to_the_framework.adoc]: Got it! Will merge this for now :)

**Pull request merged**